Source/Core/DolphinTool/VerifyCommand.cpp

```diff
@@ -75,6 +85,36 @@ int VerifyCommand::Main(const std::vector<std::string>& args)
     std::cerr << "Error: Unable to open disc image" << std::endl;
     return 1;
   }
+  const DiscIO::BlobReader& blob_reader = volume->GetBlobReader();
+
+  if (enable_block_size || enable_compression || enable_compression_level)
+  {
+    if (enable_block_size)
+    {
+      u64 block_size = blob_reader.GetBlockSize();
+      std::cout << ((block_size == 0) ? "N/A" : std::to_string(block_size)) << std::endl;
+    }
+    if (enable_compression)
+    {
+      std::string compression = blob_reader.GetCompressionMethod();
+      std::cout << ((compression == "") ? "N/A" : compression) << std::endl;
+    }
+    if (enable_compression_level)
+    {
+      u64 compression_level = blob_reader.GetCompressionLevel();
+      std::cout << ((compression_level == 0) ? "N/A" : std::to_string(compression_level))
```

Comment:
Isn't 0 a valid compression level?

Reply:
In fact, since you seem to be not doing anything with the value itself except printing it, maybe consider just returning an `std::string` in the interface? It's a bit crude but could be the best solution here?

Reply:
`std::optional<s64>` would be my first suggestion, but `std::string` could also be fine. (Do keep in mind that, if we want to show this in the GUI in the future, technically there are locales where it would be most proper to use e.g. eastern Arabic digits instead of the western Arabic digits we're used to. Not that we're doing a great job handling that in the GUI right now...)

Reply:
I've gone ahead and made the std::optional changes, but I want to also point out that zero or negative compression levels are forbidden thanks to DiscIO::GetAllowedCompressionLevels being designed for the GUI first.  Dolphin-tool rejects them, saying: "Error: Compression level not in acceptable range".  Would it be worth making a no-gui version of this function for dolphin-tool to support negative values?

Reply:
That could make sense to add. It's not a high-priority thing, though.

Regardless of whether we add the ability to use non-positive compression levels, I think we should support printing them, for the sake of supporting RVZ files made by programs other than Dolphin (and future versions of Dolphin if we change our minds later).